CHANGELOG.md

```diff
@@ -35,8 +35,19 @@
   });
   ```
 
+- Adds feedback form (beta) ([#4320](https://github.com/getsentry/sentry-react-native/pull/4328))
+
+  You can add the form component in your UI like:
+  ```jsx
+  import { FeedbackForm } from "@sentry/react-native";
+  ...
+  <FeedbackForm/>
+  ```
+  Check [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/) for more configuration options.
```

Comment:
Maybe we should remove the link to the docs until we updated them and move this out of beta

Reply:
Sounds good. Removed with 4097347e2b44db52c007a693aa9ee0efd004ca40

I'll update the [documentation PR](https://github.com/getsentry/sentry-docs/pull/11980/) as along with finalising the changes on the feedback form PRs.